README.md

```diff
@@ -9,12 +9,12 @@ This repository provides a number of CLIs interacting with a Triton
 [IMGAPI](https://github.com/joyent/sdc-imgapi) instance. There are typically
 three IMGAPI instances with which we interact:
 
-1. <https://images.joyent.com> The central repository of Joyent-vetted images
+1. <https://images.smartos.org> The central repository of Joyent-vetted images
    for using in Triton DataCenters and SmartOS machines. The `joyent-imgadm`
```

Comment:
If you update joyent-imgadm, update here too.

Reply:
Ah, thanks. I missed that.

Reply:
Also, Joyent-vetted (sorry for missing that 1st time).